### PR TITLE
Generating logs in cloud function

### PR DESCRIPTION
**Pull request by Satendra-SR**
#337 

### Types of changes

- [ ] Bug fix (a change which fixes an issue)
- [x] New feature (a change which adds functionality)

### Short description of what this resolves:

**Why these change required? What problem does it solve?**
By generating logs in a cloud function, we can get detailed information about the execution of the cloud function, which can be used to diagnose and resolve issues that arise during the execution.
  Here we are implementing the following level of logs in the system.
- INFO
- WARN
- ERROR

### Checklist:

- [x] I have performed a self-review of my own code.
- [x] The code follows the style guidelines of this project.
- [x] The code changes are passing the CI checks
- [ ] I have documented my code wherever required.
- [ ] The changes requires a change to the documentation.
- [ ] I have updated the documentation based on the my changes.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,7 +1,10 @@
 from api import services
 from flask import jsonify
 from api.helpers import db_helper
-import json
+import json, logging
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
```
Comment (GauravGusain98) [inline main.py:7]: What will happen if we dont use this line?

File: main.py [hunk 2]
```diff
@@ -22,8 +26,10 @@ def webhook(request):
                 processed = handle_payload(jsonData)
 
                 if processed is False:
+                    logger.error("[ERROR] Error processing the payload")
```
Comment (GauravGusain98) [inline main.py:29]: For testing can you simply print different level logs with some test mesaage

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]

File: main.py (hunk 2 above)
Comment (Satendra-SR) [reply main.py:29]: Okay.

File: main.py [hunk 3]
```diff
@@ -10,6 +10,9 @@ def webhook(request):
             try:
                 jsonData = request.get_json()
```
Comment (GauravGusain98) [inline main.py:11]: In this try section can you print 3 logs, 1 each for different logging level

Comment (Satendra-SR) [reply main.py:11]: Okay

**Comment by GauravGusain98** [comment]
The changes in this PR is just for testing the logs generation. All these logs will be fixed in later PR

**Pull request merged**